recipe/bld.bat

```diff
@@ -0,0 +1,34 @@
+@echo on
+setlocal EnableDelayedExpansion
+
+REM make sure we can find conda pkg installs:
+set "INCLUDE=%LIBRARY_INC%;%INCLUDE%"
+set "LIB=%LIBRARY_LIB%;%LIB%"
+
+
+REM See: https://github.com/annulen/webkit/wiki/Building-QtWebKit-on-Windows
+REM      But, use source tarball from Qt Project; projects identical as of 2018-12-14
+
+cd Tools\qt
+
+REM Debug folder containers
+dir
+
+py %SRC_DIR%\Tools\qt\build-qtwebkit-conan.py ^
+--compiler=msvc ^
+--qt=%PREFIX%\Library\include\qt ^
+--install
```

Comment:
Seems `build-qtwebkit-conan.py` was added after Alpha4 released (https://github.com/qtwebkit/qtwebkit/commits/qtwebkit-5.212/Tools/qt/build-qtwebkit-conan.py)... Maybe we need to wait for Alpha5...

Reply:
Yes, I have found that, the hard way. Either we wait for alpha5, or I go back to the hold recipes. But in those, perl and ruby don't work, not sure why.